### PR TITLE
Reducing glutin dependency in egui_glow

### PR DESCRIPTION
**Pull request by t18b219k**
glutin is just a wrapper for winit and there is no reason to use it where it is not needed.
 The team is not interested in supporting WASM.
Minimize the use of glutin, so that egui_web can be easily retired when it should be retired, e.g. when winit on wasm supports IME.

### CODE REVIEW

File: egui_glow/src/epi_backend.rs [hunk 1]
```diff
@@ -12,8 +14,8 @@ impl epi::backend::RepaintSignal for GlowRepaintSignal {
 
 #[allow(unsafe_code)]
 fn create_display(
-    window_builder: glutin::window::WindowBuilder,
-    event_loop: &glutin::event_loop::EventLoop<RequestRepaintEvent>,
+    window_builder: egui_winit::winit::window::WindowBuilder,
```
Comment (emilk) [inline egui_glow/src/epi_backend.rs:17]: I think adding `use egui_winit::winit;` at the top of the file would increase readability of this file

**Review by emilk — APPROVED** [review]
Great idea!

Some merge conflicts due to
* https://github.com/emilk/egui/pull/1154

File: egui_glow/src/epi_backend.rs (hunk 1 above)
Comment (t18b219k) [reply egui_glow/src/epi_backend.rs:17]: ok i will fix it.

File: egui_glow/src/epi_backend.rs [hunk 2]
```diff
@@ -1,8 +1,8 @@
 use crate::*;
-
+use egui_winit::winit;
 struct RequestRepaintEvent;
```
Comment (emilk) [inline egui_glow/src/epi_backend.rs:3]:
```suggestion

struct RequestRepaintEvent;
```

**Review by emilk — APPROVED** [review]

**Pull request merged**